docs/index.md

```diff
@@ -0,0 +1,178 @@
+## Amazon EKS Anywhere (EKS-A) Conformance and Validation Framework
```

Comment:
Why do we have a copy of the README here? Are we supposed to maintain both copies now? Let's find a solution to keep one copy only.